search-resources-api/src/test/java/no/unit/nva/search/SearchTicketsHandlerTest.java

```diff
@@ -5,9 +5,9 @@
 import static no.unit.nva.search.RequestUtil.DOMAIN_NAME;
```

Comment:
I might be overlooking something, but missing a test with name `shouldSetUsersTopLevelOrgAsDefaultViewingScopeIfNotSet` or something?

Reply:
Or `shouldSetCuratorsViewingScopeAsDefaultViewingScopeIfNotSet` where curators viewingScope is fetched from identity-service?